eox_tenant/settings/common.py

```diff
@@ -4,6 +4,7 @@
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+from path import Path as path
```

Comment:
Is this the reason why we had to add a new dependency in requirements @morenol ?

Reply:
Yes